scripts/history_opened_sections_history.py

```diff
@@ -2,11 +2,16 @@
 from src.reports.history_summary import HistorySummary
 
 if __name__ == "__main__":
-    years = [2014, 2015, 2016, 2017, 2018, 2019]
+    years = [2015, 2016, 2017, 2018, 2019, 2020]
 
     scout_data = ScoutData()
-    scout_data.filter_records("X_name", ["England", "Scotland", "Wales", "Northern Ireland"])
     scout_data.filter_records("Year", years)
-    history_summary = HistorySummary(scout_data)
-    history_summary.new_section_history_summary(years, report_name="opened_section_data")
+    scout_data.filter_records("X_name", ["England", "Scotland", "Wales", "Northern Ireland"])
+
+    # If filtering on IMD, remove NA values
```

Comment:
Be careful here, my preference would be to filter explicitly, there should be no 'nan' values in IMD Decile (except for the Isle of Man etc.), which have already been filtered out. Filtering out 'nan' might have unintended consequences if for example the IMD Decile didn't work.